src/lib/functions.ts

```diff
@@ -109,13 +108,17 @@ export async function trigger({
 
     res?.send?.forEach((msg) => {
       const localServerProtocols = ['ws', 'wss', 'http', 'https']
-      const serverProtocol = parsedAsyncAPI.server(msg.server).protocol().toLowerCase()
-      const isBroadcast = localServerProtocols.includes(serverProtocol) && !isRemoteServer(parsedAsyncAPI, msg.server)
+      let isBroadcast: boolean = true
```

Comment:
You don't need the type there:

```suggestion
      let isBroadcast = true
```